tests/_support/Page/AbstractPage.php

```diff
@@ -58,26 +58,51 @@ class AbstractPage
 	 */
 	public static $buttonInstall = 'Install';
 
+	/**
+	 * @var string
+	 */
+	public static $tabTranslations = '//ul[@id="configTabs"]/li[2]/a';
+
 	/**
 	 * @var string
 	 */
 	public static $chooseNo = 'No';
 
+	/**
+	 * @var string
+	 */
+	public static $tabWebServices = '//ul[@id="configTabs"]/li[3]/a';
+
 	/**
 	 * @var string
 	 */
 	public static $chooseYes = 'Yes';
 
+	/**
+	 * @var string
+	 */
+	public static $id = '//ul[@id="configTabs"]/li[2]/a';
+
 	/**
 	 * @var string
 	 */
 	public static $amAdmin = 'administrator';
 
+	/**
+	 * @var string
+	 */
+	public static $savemess = 'iv[@id="system-message-container"]';
+
 	/**
 	 * @var string
 	 */
 	public static $labelAdmin = 'Administrator';
 
+	/**
+	 * @var string
+	 */
+	public static $title = 'redCORE Config';
```

Comment:
Please use title on page redCOREConfigPage https://github.com/redCOMPONENT-COM/redCORE/blob/develop/tests/_support/Page/redCOREConfigPage.php#L24

Reply:
please check it again